style.css

```diff
@@ -65,7 +64,7 @@ section {
   padding: 0 20px;
 }
```

Comment:
Kindly adjust the space between the headline section and the works section in order to match the design template. You have more space than the design does.

Your design:
![image](https://user-images.githubusercontent.com/20554583/219922172-5f9f7185-b894-4d79-9f19-9de65c3843bc.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/219922176-92f9c3c4-1a13-4dbd-a37f-187f50436c60.png)

Reply:
This one too, on the full web page looks normal, So I think I miss something in the lecture, maybe I will try to find the right way to fit it, but on the full webpage, it works well.